README.md

```diff
@@ -0,0 +1,120 @@
+# NgsiLdToCkan
+
+[![FIWARE](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/core.svg)](https://www.fiware.org/developers/catalogue/)
+[![NGSI-LD badge](https://img.shields.io/badge/NGSI-LD-red.svg)](https://www.etsi.org/deliver/etsi_gs/CIM/001_099/009/01.06.01_60/gs_CIM009v010601p.pdf)
+[![License: Apache-2.0](https://img.shields.io/github/license/stellio-hub/stellio-context-broker.svg)](https://spdx.org/licenses/Apache-2.0.html)
+![Build](https://github.com/easy-global-market/nifi-ngsild-ckan/actions/workflows/maven.yml/badge.svg)
+
+## Table of Contents
+- [Overview](#overview)
+- [Functionality](#functionality)
+   - [CKAN Data Structures And Mapping](#ckan-data-structures-and-mapping)
+   - [Input Data](#input-data)
+   - [Output](#output)
+- [Configuration](#configuration)
+   - [Processor Properties](#processor-properties)
+- [Metadata](#metadata)
+   - [Metadata List](#metadata-list)
+
+## Overview
+`NgsiLdToCkan` is a Nifi processor developed by FIWARE to persist NGSI-LD context data events within a [CKAN](https://ckan.org/) server. Context data is received as a notification sent by a Context Broker or any system that supports NGSI-LD.
+
+
+## Functionality
+
+The processor receives NGSI-LD events (notifications) and converts them internally into an `NGSIEvent` objects, which contain the entities to be published on the CKAN server.
+
+### CKAN Data Structures And Mapping
```

Comment:
here it really is missing something that explains the expected NGSI-LD representation of entities to be received, with the `Dataset` and `Distribution` types. it is also missing a reference to the DCAT standard.

Reply:
and also explains there is a pending point about CKAN organizations for which we don't yet have a clear solution / mapping (in the context broker, we mainly have `Catalog` entities which is different). Thus the `X-CKAN-OrganizationName` added in subscriptions (btw, they should explained, without subscriptions, the thing does not work)